test/integration/spec/rtcpeerconnection.js

```diff
@@ -135,7 +131,7 @@ describe('RTCPeerConnection', function() {
       const constraints = { audio: true, video: true };
       const stream = await makeStream(constraints);
       const pc = new RTCPeerConnection({ iceServers: [] });
-      pc.addStream(stream);
+      stream.getTracks().forEach(track => pc.addTrack(track, stream));
```

Comment:
Since this is repeated, maybe a helper function,

```js
function addStream(pc, stream) {
  stream.getTracks().forEach(track => pc.addTrack(track, stream));
}
```

could DRY this up?